www/blackberry10/README.md

```diff
@@ -0,0 +1,75 @@
+# Facebook Requirements and Set-Up [Blackberry10]
+
+To use this plugin you will need to make sure you've registered your Facebook app with Facebook and have an `APP_ID` [https://developers.facebook.com/apps](https://developers.facebook.com/apps).
+
+This plugin was based on a Micro-Library for facebook called [openFB] (https://github.com/ccoenraets/OpenFB)
+
+## Install
+
+This plugin requires [Cordova CLI](http://cordova.apache.org/docs/en/4.0.0/guide_cli_index.md.html).
+
+To install the plugin in your app, execute the following (replace variables where necessary):
+
+```sh
+# Create initial Cordova app
+$ cordova create myApp
+$ cd myApp/
+$ cordova platform add browser
+
+# Remember to replace APP_ID and APP_NAME variables
+$ cordova plugin add https://github.com/Wizcorp/phonegap-facebook-plugin/ --variable APP_ID="123456789" --variable APP_NAME="myApplication"
```

Comment:
Probably should put the BB10 fork instead of the way upstream one, as this is where the code will live for some unknown time.